bde_env.drush.inc

```diff
@@ -82,7 +82,18 @@ function drush_bde_env_gen($filename = '') {
     ),
   );
 
-  $output = 'export BEHAT_PARAMS=' . escapeshellarg(json_encode($parameters, JSON_UNESCAPED_SLASHES));
+  switch (getenv('SHELL')) {
+    case '/usr/bin/fish':
```

Comment:
You're correct, a better approach would be to explode the SHELL environment variable by '/' and then just look into the last item of the array.